src/mbed_tools/devices/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""API to detect any Mbed OS devices connected to the host computer.
+
+It is expected that this package will be used by developers of Mbed OS tooling rather than by users of Mbed OS.
+This package uses the https://github.com/ARMmbed/mbed-targets interface to identify valid Mbed Enabled Devices.
+Please see the documentation for mbed-targets for information on configuration options.
+
+For the command line interface to the API see the package https://github.com/ARMmbed/mbed-tools
+"""
+from mbed_tools.devices.devices import get_connected_devices
```

Comment:
Does this need to be "devices.devices" or is there a nicer or flatter name we can use?

Reply:
`devices.get_devices`? We're importing to the package level here in `__init__.py`, so calling clients will import `devices.get_connected_devices` but I agree it looks noisy.

Reply:
OK